examples/swig_boids_spatial3D/boids_spatial3D.py

```diff
@@ -165,7 +165,7 @@ def clampPosition(x, y, z, MIN_POSITION, MAX_POSITION):
 // Agent function
 FLAMEGPU_AGENT_FUNCTION(inputdata, flamegpu::MessageSpatial3D, flamegpu::MessageNone) {
     // Agent properties in local register
-    const flamegpu::id_t id = FLAMEGPU->getID();
+    const int id = FLAMEGPU->getID();
```

Comment:
Why have you changed this from `id_t`, which the method returns? 

Looks like it might have been to standardize it with the C++ implementations, though it should have gone the other way (in case the id_t typedef changes in the future to be a wider integer type)

Reply:
Accidental change when copy-pasting. I'll revert this and change the others to id_t as well.